packages/components/src/components/Dropzone/Dropzone.tsx

```diff
@@ -0,0 +1,234 @@
+  isFocused: boolean,
+  isDragAccept: boolean,
+  isDragReject: boolean
+): {
+  backgroundColor?: SystemProp<keyof Theme["colors"], Theme>;
+  borderColor?: SystemProp<keyof Theme["colors"], Theme>;
+} => {
+  if (isLoading || isDragAccept) {
+    return {
+      backgroundColor: "colorBackgroundInfo",
+      borderColor: "colorBorderPrimary",
+    };
+  }
+  if (successState) {
+    return {
+      backgroundColor: "colorBackgroundSuccess",
+      borderColor: "colorBorderSuccess",
+    };
+  }
+  if (errorState || isDragReject) {
+    return {
+      backgroundColor: "colorBackgroundError",
+      borderColor: "colorBorderError",
+    };
+  }
+  if (isFocused) {
+    return {
+      backgroundColor: "colorBackground",
+      borderColor: "colorBorderPrimary",
+    };
+  }
+  return {
+    backgroundColor: "colorBackgroundWeakest",
+    borderColor: "colorBorder",
+  };
+};
+
+const getIcon = (
+  defaultState: boolean,
+  errorState: string | undefined,
+  isLoading: boolean,
+  successState: boolean,
+  isDragReject: boolean
+) => {
+  let icon: IconProps["icon"] = "CloudArrowUpIcon";
+  let iconColor: IconProps["color"] = "colorIconInfo";
+  if (successState) {
+    icon = "CheckCircleIcon";
+    iconColor = "colorIconSuccess";
+  }
+  if (isDragReject || errorState) {
+    icon = "ExclamationTriangleIcon";
+    iconColor = "colorIconError";
+  }
+
+  return <Icon color={iconColor} decorative icon={icon} size={"sizeIcon40"} />;
+};
+
+const Dropzone = React.forwardRef<HTMLDivElement, DropzoneProps>(
+  (
+    {
+      onCancelClick,
+      error,
+      fileTypes = { "application/pdf": [".pdf"] },
+      maxFileSize = MAX_FILE_SIZE,
+      onDrop,
+      progress,
+    },
+    ref
+  ) => {
+    const [dropZoneError, setDropZoneError] = useState(error);
+    const defaultState = !dropZoneError && !progress;
+    const errorState = dropZoneError;
+    const isLoading = !!progress && progress > 0 && progress < 100;
+    const successState = progress === 100 && !dropZoneError;
+
+    // console.log("defaultState", defaultState);
+    // console.log("errorState", errorState);
+    // console.log("isLoading", isLoading);
+    // console.log("successState", successState);
+    const {
+      acceptedFiles,
+      fileRejections,
+      getRootProps,
+      getInputProps,
+      isDragActive,
+      isDragAccept,
+      isDragReject,
+      isFocused,
+    } = useDropzone({
+      maxFiles: 1,
+      maxSize: maxFileSize,
+      multiple: false,
+      accept: {
+        ...fileTypes,
+      },
+      onDropAccepted: () => {
+        onDrop(acceptedFiles);
+      },
+      onDropRejected: (fileRejections) => {
+        forEach(fileRejections, (file) => {
+          forEach(file.errors, (err) => {
+            if (err.code === "file-too-large") {
+              setDropZoneError("File must be less than 50mb.");
+            }
+            if (err.code === "file-invalid-type") {
+              setDropZoneError("Wrong file type. PDF format only.");
```

Comment:
Since we're allowing multiple file types, we may want to reword this error. We are allowing multiple file types right?